dpub/core.py

```diff
@@ -95,21 +128,27 @@ def _read_tests_map(drive,
     ids = drive.read(doc, t_range, read_dimension)
 
     tests_map = {}
+    empty_ids = 0
```

Comment:
There's another way to identify the row for writting. All the sheet test have a cell where writting the first trace. That cell has a row. The max row is the last one. Said this, this other is not that bad either, though more verbose

Reply:
Yes, I did. This variable isn't longer exist.